crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/generated_acir.rs

```diff
@@ -680,6 +664,30 @@ impl GeneratedAcir {
         Ok(())
     }
 
+    /// Adds a directive which injects witnesses with values `q = a / b` and `r = a % b`.
```

Comment:
nit: We probably want to note that we add constraints to ensure that quotient and remainder are range constrained